modules/installation-special-config-encrypt-disk.adoc

```diff
@@ -0,0 +1,155 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-encrypting-disks_{context}"]
+= Encrypting disks during installation
+
+During {product-title} installation, you can enable disk encryption on all master and worker nodes.
+This feature:
+
+* Is available for IPI and bare metal {product-title} deployments
+* Is supported on Red Hat CoreOS (RHCOS) systems only
+* Sets up disk encryption during the manifest installation phase so all data written to disk, from first boot forward, is encrypted
+* Encrypts data on the root filesystem only (/dev/mapper/coreos-luks-root on /)
+* Requires no user intervention for providing passphrases
+* Uses AES-256-CBC encryption
+* Should be enabled for your cluster to be FIPS compliant.
+See link:https://docs.openshift.com/container-platform/4.3/installing/installing-fips.html[FIPS compliant {product-title} clusters]
```

Comment:
xref:../installing/installing-fips.adoc#installing-fips[FIPS compliant {product-title} clusters]

Reply:
I removed the link from this module and added it into the end of the assembly (installing-customizing.adoc) to the "Additional resources" section:

See xref:../installing/installing-fips.adoc#installing-fips[FIPS compliant {product-title} clusters]
for information on FIPS compliance.

Reply:
I had to go back and fix it. I missed an extra ../ at the beginning of the xref. Fixed now.